renovate.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I would like to reuse the config, as in https://github.com/kubewarden/verify-image-signatures/blob/main/renovate.json.

Reply:
done